rust/src/error.rs

```diff
@@ -0,0 +1,3 @@
+pub type Error = anyhow::Error;
+pub type Result<T> = std::result::Result<T, Error>;
```

Comment:
Can you just `pub use anyhow::Result;`?

Reply:
Yeah, I might just move this into main under `mod error` and `pub use` both Error and Result.